packages/suite/src/views/settings/coins/index.tsx

```diff
@@ -47,6 +51,14 @@ const Settings = () => {
 
     return (
         <SettingsLayout>
+            {isDeviceLocked && (
+                <DeviceBanner
```

Comment:
I added a comment to the original issue. On small screens, it looks bad. Probably some padding on the right and using `font-weight: 500;` instead of `600` could help.
![Screenshot 2022-01-24 at 13 54 19](https://user-images.githubusercontent.com/33235762/150787874-c244fb29-dfcf-47b4-bf1c-cc7e7115fb08.png)

Reply:
![Screenshot 2022-01-24 at 14 05 49](https://user-images.githubusercontent.com/33235762/150788054-76398399-d5b6-4a5d-97f9-3165306a4e57.png)

Reply:
99a7501

Reply:
`font-weight` not changed